.github/workflows/ci.yml

```
@@ -29,7 +29,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.11"]
+        python-version: ["3.7", "3.12"]
```

```suggestion
        python-version: ["3.7", "3.11"]
```

Will update min test later.